src/index.js

```diff
@@ -46,10 +46,11 @@ class MCServer extends EventEmitter {
 
   connect (options) {
     const version = require('minecraft-data')(options.version).version
-    if (!supportedVersions.some(v => v.includes(version.majorVersion))) {
+    if (!supportedVersions.some(v => v.includes(version.minecraftVersion))) {
```

Comment:
I recommend we replace that by the same system as in mineflayer https://github.com/PrismarineJS/mineflayer/blob/master/lib/loader.js#L111-L120

Reply:
That seems reasonable.

Reply:
The link you sent was for ensuring that the client and server are using compatible protocols, not for ensuring that the version is supported by mineflayer.

Reply:
```
    const versionData = bot.registry.version
    if (versionData['>'](latestSupportedVersion)) {
      throw new Error(`Server version '${serverPingVersion}' is not supported. Latest supported version is '${latestSupportedVersion}'.`)
    } else if (versionData['<'](oldestSupportedVersion)) {
      throw new Error(`Server version '${serverPingVersion}' is not supported. Oldest supported version is '${oldestSupportedVersion}'.`)
    }
```
this is what checks supported versions in mineflayer